scripts/release-current-platform.js

```diff
@@ -74,42 +96,55 @@ function copyFile(sourceFile, destFile) {
   });
 }
 
+/**
+ * Convenience function that copies the generated prebuild
+ * binaries for the current platform into a temporary folder
+ * with the correct name (name that contains platform, arch, abi
+ * version, and/or gitsha).
+ *
+ * @returns Promise
+ */
 function prepareBinaries() {
-  const promises = [];
   for (var i = 0 ; i < archs.length ; i++) {
     const arch = archs[i];
 
-    const abiPromises = builtVersions.map( runtime => {
+    builtVersions.map( runtime => {
       const abi = [runtime].join('-');
       const nodeFileNames = [abi, version, platform, arch];
       if (sha.length > 0) {
         nodeFileNames.push(sha);
       }
       const sourceFile = path.join('.', 'prebuilds', [platform, arch].join('-'), [abi, 'node'].join('.'));
-      const destFile = path.join('.', 'dist', [nodeFileNames.join('-'), 'node'].join('.'))
-      filesToSend.push(destFile);
-      return copyFile(sourceFile, destFile);
+      const tmpDestFile = path.join('.', 'tmp', [nodeFileNames.join('-'), 'node'].join('.'))
+      filesToSend.push(tmpDestFile);
+      return copyFile(sourceFile, tmpDestFile);
     });
-    promises.push(Promise.all(abiPromises));
-
   }
-  return Promise.all(promises);
+  return Promise.resolve();
 }
 
+/**
+ * Prepares the dist folder with all necessary files such as
+ * the compiled source code of device-api-usb, package.json,
+ * manifest.json and the scripts directory used during install.
+ * This is useful when referencing device-api-usb dependency
+ * using the .tar.gz file.
+ *
+ * @returns Promise
+ */
 function prepareDistPackage() {
   fs.mkdirSync(destDir);
+  fs.mkdirSync(destScripts);
 
   delete pkg.devDependencies;
   delete pkg.optionalDependencies;
   delete pkg.bundledDependencies;
-  pkg.scripts = {
-    install: pkg.scripts.install_released_version,
-  };
-
   console.log('Copy files', destDir);
+  console.log('Copy scripts', destScripts);
   return Promise.all(
     [
-      copyDirectory('./lib/**/*', path.join(destDir, 'lib')),
+      copyDirectory('./lib/**/*', path.join(destDir)),
```

Comment:
This should end up in lib not on root otherwise the paths can't be resolved.

Reply:
Good catch